### PR TITLE
Refactor MTE-1375 [v118] Disable animations when running XCUITests

### PR DESCRIPTION
**Pull request by isabelrios**
There are tests taking more than 2 mins to run because they stuck on this screen, Action Menu, waiting for the animation to finish.
Simulator Screenshot - iPhone 14 - 2023-08-01 at 11 53 46

With this PR we are disabling animations so that tests run much faster.

### CODE REVIEW

**Review by dragosb01 — APPROVED** [review]

File: Tests/XCUITests/BaseTestCase.swift [hunk 1]
```diff
@@ -37,7 +37,9 @@ class BaseTestCase: XCTestCase {
                            LaunchArguments.DeviceName,
                            "\(LaunchArguments.ServerPort)\(serverPort)",
                            LaunchArguments.SkipContextualHints,
-                           LaunchArguments.TurnOffTabGroupsInUserPreferences]
+                           LaunchArguments.TurnOffTabGroupsInUserPreferences,
+                           LaunchArguments.DisableAnimations
```
Comment (clarmso) [inline Tests/XCUITests/BaseTestCase.swift:41]: 🎉 🎉 🎉

**Comment by clarmso** [comment]
@isabelrios Have you tried to run the full functional tests using the `LaunchArguments.DisableAnimations` argument?

**Comment by isabelrios** [comment]
> @isabelrios Have you tried to run the full functional tests using the `LaunchArguments.DisableAnimations` argument?

they will use it as the smoketest because it is defined in the BaseTestCase, but I will start the run locally to be sure they work before merging

**Comment by isabelrios** [comment]
> > @isabelrios Have you tried to run the full functional tests using the `LaunchArguments.DisableAnimations` argument?
> 
> they will use it as the smoketest because it is defined in the BaseTestCase, but I will start the run locally to be sure they work before merging

This is working on my end

**Review by clarmso — APPROVED** [review]

**Pull request merged**